performance_test.py

```diff
@@ -0,0 +1,35 @@
+import json
+
+
+def test_create_testing_files():
+    i = 0
+    lots_declarations = []
+    samples_declarations = []
```

Comment:
Maybe rename `samples_declarations` to `samples` as the data appended is not for the `samples_declarations` collection but the `samples` collection
And possibly `lots_declarations` to reference  `samples_declarations` ?